lib/mayaUsd/nodes/layerManager.cpp

```diff
@@ -600,7 +614,8 @@ std::string LayerDatabase::getSelectedStage() const { return _selectedStage; }
 
 bool LayerDatabase::saveLayerManagerSelectedStage()
 {
-    MayaUsd::LayerManager* lm = findOrCreateNode();
+    const MFnReference*    fromReference = nullptr;
+    MayaUsd::LayerManager* lm = findOrCreateNode(fromReference);
```

Comment:
Depends on personal preferences, I prefer to be explicit, but both ways are fine. Initially, all functions (like saveLayerManagerSelectedStage) received fromReferences, but I realized that the save functions are never saving Maya references. I'll change it since I need to change the copyright anyway.